RestService/Client.php

```diff
@@ -120,6 +120,23 @@ public function getController()
         return $this->controller;
     }
 
+    /**
+     * @param string $out
+     */
+    protected function output($out){
+    	if( !ini_get('zlib.output_compression') && 'ob_gzhandler' != ini_get('output_handler') && isset($_SERVER['HTTP_ACCEPT_ENCODING']) ){
+    		header('Vary: Accept-Encoding'); // Handle proxies
+    		if( false !== stripos($_SERVER['HTTP_ACCEPT_ENCODING'], 'deflate') && function_exists('gzdeflate') ){
```

Comment:
This condition is the [YODA condition](https://en.wikipedia.org/wiki/Yoda_conditions).

Does this package use YODA condition?